service/src/main/java/uk/gov/hmcts/reform/fpl/service/sdo/UrgentGatekeepingOrderService.java

```diff
@@ -71,6 +71,7 @@ public GatekeepingOrderEventData prePopulate(CaseData caseData) {
         return returnedData;
     }
 
+    @Deprecated
```

Comment:
why is this deprecated?

Reply:
same method is in GatekeepingOrderService and this one is only used in old StandardDirectionController which will be removed